.github/workflows/base-ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     container: baoproject/bao:latest
     steps:
       - uses: actions/checkout@v2
-      - run: | 
+      - run: |
```

Comment:
Beware of this kind of change. Removing this kind of trailing white spaces or similar issues should be done in separate commits.